src/nvim/ex_getln.c

```diff
@@ -1602,12 +1602,16 @@ static int command_line_changed(CommandLineState *s)
              && cmd_can_preview(ccline.cmdbuff)
              && !vpeekc_any()) {
     // Show 'inccommand' preview. It works like this:
+    //    0. Block error reporting since the command may not yet be functional
```

Comment:
this and its counterpart seems like an implementation detail, maybe inline it next to the respective increment/decrement.